h/search/index.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, session, es_client, request, target_index=None, op_type='inde
         else:
             self._target_index = target_index
 
-    def index(self, annotation_ids=None):
+    def index(self, annotation_ids=None, windowsize=PG_WINDOW_SIZE, chunk_size=ES_CHUNK_SIZE):
```

Comment:
Could you update the docstring for this method to explain what the `windowsize` and `chunk_size` parameters are for.